src/main/java/com/easypost/model/beta/Referral.java

```diff
@@ -115,15 +114,12 @@ public static List<Referral> all(final Map<String, Object> params) throws EasyPo
      *
      * @param params Map of parameters.
      * @param apiKey API key to use in request (overrides default API key).
-     * @return List<Referral> object.
+     * @return ReferralCustomerCollection object.
      * @throws EasyPostException when the request fails.
      */
-    public static List<Referral> all(final Map<String, Object> params, String apiKey) throws EasyPostException {
-        Referral[] response =
-                request(RequestMethod.GET, String.format("%s/%s", EasyPost.BETA_API_BASE, "referral_customers"), params,
```

Comment:
The reason it didn't work for you is because of this bad copy/paste that altered what endpoint it hit. Tweak this back to what it's supposed to be and you should be golden.